### PR TITLE
Updating README.md to include @HensonNavigable and Parceler integration

### PR DESCRIPTION
**Pull request by dlemures**
Some changes to the README.md to get ready for 2.0!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,28 +58,82 @@ User user = Dart.get(bundle, "key"); // User implements Parcelable
...
 * either the package specified by the `dart.henson.package` annotation processor option
 * or if no such option is used, in the common package of all annotated activities. See the Javadoc of HensonExtraProcessor for more details.
 
+Bonus
+-----
+As you can see from the examples above, using both Dart & Henson not only provided a very structured generated navigation layer and conveninent DSLs, it also completely transparently allows to wrap/unwrap parcelables.
+
+Parceler
+-------------------------
+Dart 2.0 comes with an out-of-the-box integration with [Parceler](https://github.com/johncarl81/parceler).
```
Comment (stephanenicolas) [inline README.md]: offers a built-in support for Parceler. Using Parceler with Dart 2 is optional.

If you use Parceler, Dart will automatically detect..

File: README.md [hunk 2]
```diff
@@ -58,28 +58,82 @@ User user = Dart.get(bundle, "key"); // User implements Parcelable
...
+```java
+Intent intent = Henson.with(this)
+        .gotoOneMoreActivityActivity()
+        .extra(new ParcelExample())
+        .build();
+
+startActivity(intent);
+```
+
+This integration is optional and will only happen if Parceler is included inside the project.
```
Comment (stephanenicolas) [inline README.md]: Parceler usage is optional and will take place only when Parceler is present in the classpath.

**Comment by stephanenicolas** [comment]
Comments + RTM

File: README.md (hunk 1 above)
Comment (dlemures) [inline README.md]: updated

File: README.md (hunk 2 above)
Comment (dlemures) [inline README.md]: updated

**Pull request merged**